source/compatibility.tex

```diff
@@ -1289,7 +1289,7 @@
 \ref{istream::sentry},
 \ref{ostream::sentry},
 \ref{iostate.flags}
-\change Specify use of explicit in existing boolean conversion operators
+\change Specify use of explicit in existing boolean conversion functions
```

Comment:
OK, sure, but I would have had no problem with a minor drive-by fix like that if this is a single, isolated problem. You can stick it into a separate commit in the same pull request, and I could retain the separate commits (i.e. not squash them), so we still get nicely atomic commits.